2-mandatory/3-stocks.js

```diff
@@ -35,8 +35,25 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
 */
 function getAveragePrices(closingPricesForAllStocks) {
     // TODO
+
+   let output=[]
+
+    for (let i=0 ; i< closingPricesForAllStocks.lenght; i++){
+        const price = closingPricesForAllStocks[i];
+        const totalPrice= price.reduce(function (a,b){
+            return a+b;
+        }, 0)
+        const avarage = totalPrice / price.lenght;
+        const  avarageFixed=avarage.toFixed(2)
+
+        output.push(avarageFixed)
 }
```

Comment:
Hi Baki
I sew you done nice work here with big effort and I want add here you can use for loop in two time instead to use two function 

check here 

```
function getAveragePrices(closingPricesForAllStocks) {
    let result = [];
    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
    let total = 0;
    for (let j = 0; j < closingPricesForAllStocks[i].length; j++) {
        total += closingPricesForAllStocks[i][j];
    }
    result.push(parseFloat((total / 5).toFixed(2)));
    }
    return result;
}
```

this is my review for this week 
Thanks Baki